lib/commands.coffee

```diff
@@ -6,6 +6,7 @@ class Commands
     @_subscriptions.add atom.commands.add 'atom-workspace',
       'linter:next-error': @nextError.bind(@)
       'linter:toggle': @toggleLinter.bind(@)
+      'linter:set-bubble-transparent': => @setBubleTransparent()
```

Comment:
That's fine. Both are valid options but this become less clean if parameters were involved.

```CoffeeScript
    'linter:do-something-with-parameters': (param1, param2, param3) => @something(param1, param2, param3)
```

I favor the fat arrow on the target because it's always clean.